TezosKit/Client/TezosClient.swift

```diff
@@ -556,7 +555,7 @@ public class TezosClient {
       urlRequest.httpBody = payloadData
     }
 
-    let request = urlSession.dataTask(with: urlRequest as URLRequest) { data, response, error in
+    let request = urlSession.dataTask(with: urlRequest as URLRequest) { [callbackQueue] data, response, error in
```

Comment:
Did you mean [weak callbackQueue] ?

Or better yet just use [weak self] and return early if setting strongSelf fails?

Reply:
Done. I used `self` instead of `strongSelf` for the sake of consistency with the rest of the file.